forager_backend/forager_backend_api/views.py

```diff
@@ -529,13 +529,13 @@ def process_image_query_results(request, dataset, query_response):
     filtered_pks = filtered_images(request, dataset)
     # TODO(mihirg): Eliminate this database call by directly returning pks from backend
     dataset_items = DatasetItem.objects.filter(pk__in=filtered_pks)
-    dataset_items_by_path = {di.path: di for di in dataset_items}
+    dataset_items_by_identifier = {di.identifier: di for di in dataset_items}
```

Comment:
This is because the embedding server was updated but this endpoint wasn't?

Reply:
this is a utility function that is involved in every model-based query, but yes exactly